README.md

```diff
@@ -26,6 +26,12 @@ Note on Windows: this step requires permission to create symlinks. Either run th
 dart run arcgis_maps install
 ```
 
+Use `build_runner` to generate support code.
+
+```
+dart run build_runner build
+```
+
```

Comment:
I guess I didn't realise this would be another step for our users which I'm on the fence about. Not totally opposed but would prefer not since the benefits of doing it is really to aid our team. 
Could we maybe have a single command for the samples that does the install and build steps?

Reply:
We could present it as:

```sh
dart run arcgis_maps install && dart run build_runner build
```

And/or we could drop in a script that does those two steps (as well as `flutter pub upgrade`). Call it `initialize.sh` maybe.

Reply:
ok, I added a script and edited the README

Reply:
Nice. My only thought now is if we should call that shell/bat from a dart script so our users call `dart run initialize` or something. Can you raise this in parking lot - I'm interested in thoughts from others in the team on this experience. It's quite a key entry point so I just want to give it the thought and discussion.

Reply:
`dart tool/initialize.dart`

Reply:
I think this works well. 
My only query now before approving is whether for a publicly consumed script it should be in `bin` rather than an internal tool. So it would be like: 
`dart run arcgis_maps_sdk_flutter_samples initialize`
I guess the lines are a little blurred. Interested in your thoughts.

Reply:
`dart run arcgis_maps_sdk_flutter_samples initialize` is maybe a little more descriptive of what it's doing too as side effect.

Reply:
I think this is definitely an internal tool. If you're building the Sample Viewer, you are stepping inside the project to build it (and the project is otherwise useless until this is run). The `bin` folder would be for functionality that is being provided for some purpose outside of the project.